modules/reference/pages/rpk/rpk-topic/rpk-topic-create.adoc

```diff
@@ -2,17 +2,10 @@
 
 Create topics.
 
-All topics created with this command will have the same number of partitions,
+Topics created with this command will have the same number of partitions,
 replication factor, and key/value configs.
 
-For example:
-
-[,bash]
-----
-rpk topic create -c cleanup.policy=compact -r 3 -p 20 foo bar
-----
-
-This example command creates two topics, `foo` and `bar`, each with 20 partitions, 3 replicas, and the `cleanup.policy=compact` config option set.
+For more information about topics, see xref:reference:rpk/rpk-topic/rpk-topic-describe.adoc[`rpk topic describe`].
 
 == Usage
```

Comment:
@Deflaimun similar comment as in `rpk topic consume`: Can you put the defaults in a separate sentence, like: ". Default is -1."

Reply:
I'll create a separate issue for it, since it's a global problem in our website docs for rpk.

Reply:
https://github.com/redpanda-data/documentation-private/issues/2286